cypress/support/e2e.ts

```diff
@@ -10,11 +10,11 @@ async function main() {
     await installChainlit();
   }
 
-  if (singleTest) {
-    await runTest(singleTest);
-  } else {
+  // if (singleTest) {
```

Comment:
do we still support single test runs?

Reply:
It is of course planned to restore this behavior :) I think we can even add regex support without effort.